.github/workflows/packages-bin.yml

```
@@ -218,6 +219,10 @@ jobs:
         pyver: [cp38, cp39, cp310, cp311, cp312, cp313]
 
     steps:
+      # there are some other libpq in PATH
+      - run: rm c:/tools/php C:/Strawberry/c/bin -rf
+        shell: bash
```

This order is not the standard way to use `rm`. I also don't think that `shell: bash` is required.

```suggestion
      - run: rm -rf c:/tools/php C:/Strawberry/c/bin
```

---

Weird: it's not required in `tests.yml`.

---

I see. Probably worth uniforming them... But we can leave it out of this MR.